GitUI/CommandsDialogs/FormDeleteBranch.cs

```diff
@@ -1,24 +1,25 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Diagnostics;
+using System.Drawing;
 using System.Linq;
 using System.Windows.Forms;
+using GitCommands;
 using GitCommands.Git;
 using GitCommands.Git.Commands;
+using GitExtUtils.GitUI;
 using GitUIPluginInterfaces;
 using ResourceManager;
 
 namespace GitUI.CommandsDialogs
 {
-    public sealed partial class FormDeleteBranch : GitModuleForm
+    public sealed partial class FormDeleteBranch : GitExtensionsDialog
     {
         private readonly TranslationString _deleteBranchCaption = new("Delete branches");
-        private readonly TranslationString _deleteBranchQuestion = new(
-            "Are you sure you want to delete selected branches?" + Environment.NewLine + "Deleting a branch can cause commits to be deleted too!");
-        private readonly TranslationString _deleteUnmergedBranchForcingSuggestion =
-            new("You cannot delete unmerged branch until you set “force delete” mode.");
         private readonly TranslationString _cannotDeleteCurrentBranchMessage =
             new("Cannot delete the branch “{0}” which you are currently on.");
+        private readonly TranslationString _deleteBranchConfirmTitle = new("Delete Confirmation");
```

Comment:
❔
```suggestion
        private readonly TranslationString _deleteBranchConfirmTitle = new("Delete confirmation");
```

Reply:
This is consistent with other titles. If we want to change them it needs to be done separately across the codebase.

Reply:
How should we capitalize window titles?
At least for FormDeleteBranch and its TaskDialog it should be consistent (see screenshots), shouldn't it?